tools/components.yaml

```diff
@@ -0,0 +1,55 @@
+- Key: ABSDataBus
```

Comment:
Not ideal to duplicate it here and maintain a 2nd copy.
Perhaps what we've discussed during Azure maintainers sync would be the end goal - have this under docs.